graphqlapi/src/main/resources/playground.html

```diff
@@ -0,0 +1,586 @@
+      <path class="cWrBmb" x="95.97162628173828" y="83.4900016784668" fill="#fff"
+            d="M102.9 87.5c-2.2 3.84-7.1 5.15-10.94 2.94-3.84-2.2-5.14-7.12-2.94-10.96 2.2-3.84 7.12-5.15 10.95-2.94 3.86 2.23 5.16 7.12 2.94 10.96"
+            style="transform: translate(100px, 100px);"></path>
+      <path class="Wnusb" x="64" y="101.97999572753906" fill="#fff"
+            d="M64 110c-4.43 0-8-3.6-8-8.02 0-4.44 3.57-8.02 8-8.02s8 3.58 8 8.02c0 4.4-3.57 8.02-8 8.02"
+            style="transform: translate(100px, 100px);"></path>
+      <path class="bfPqf" x="32.03982162475586" y="83.4900016784668" fill="#fff"
+            d="M25.1 87.5c-2.2-3.84-.9-8.73 2.93-10.96 3.83-2.2 8.72-.9 10.95 2.94 2.2 3.84.9 8.73-2.94 10.96-3.85 2.2-8.74.9-10.95-2.94"
+            style="transform: translate(100px, 100px);"></path>
+      <path class="edRCTN" x="32.033552169799805" y="46.510000228881836" fill="#fff"
+            d="M38.96 50.52c-2.2 3.84-7.12 5.15-10.95 2.94-3.82-2.2-5.12-7.12-2.92-10.96 2.2-3.84 7.12-5.15 10.95-2.94 3.83 2.23 5.14 7.12 2.94 10.96"
+            style="transform: translate(100px, 100px);"></path>
+      <path class="iEGVWn" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M63.55 27.5l32.9 19-32.9-19z"></path>
+      <path class="bsocdx" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M96 46v38-38z"></path>
+      <path class="jAZXmP" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M96.45 84.5l-32.9 19 32.9-19z"></path>
+      <path class="hSeArx" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M64.45 103.5l-32.9-19 32.9 19z"></path>
+      <path class="bVgqGk" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M32 84V46v38z"></path>
+      <path class="hEFqBt" stroke="#fff" stroke-width="4" stroke-linecap="round"
+            stroke-linejoin="round" d="M31.55 46.5l32.9-19-32.9 19z"></path>
+      <path class="dzEKCM" id="Triangle-Bottom" stroke="#fff" stroke-width="4" d="M30 84h70"
+            stroke-linecap="round"></path>
+      <path class="DYnPx" id="Triangle-Left" stroke="#fff" stroke-width="4" d="M65 26L30 87"
+            stroke-linecap="round"></path>
+      <path class="hjPEAQ" id="Triangle-Right" stroke="#fff" stroke-width="4" d="M98 87L63 26"
+            stroke-linecap="round"></path>
+    </g>
+  </svg>
+  <div class="text">Loading
+    <span class="dGfHfc">GraphQL Playground</span>
+  </div>
+</div>
+
+<div id="root"/>
+<script type="text/javascript">
+    window.addEventListener('load', function (event) {
+
+      const loadingWrapper = document.getElementById('loading-wrapper');
+      loadingWrapper.classList.add('fadeOut');
+
+      const root = document.getElementById('root');
+      root.classList.add('playgroundIn');
+
+      urlParams = new URLSearchParams(window.location.search);
```

Comment:
`UrlSearchParams` is [modern-ish](https://caniuse.com/?search=URLSearchParams) so some browsers might not support it.

JavaScript allows `typeof undeclaredVariable` to check if there's a global function/class defined.

In this case, it should be something like:

```javascript
let tokenValue = null;
if (typeof URLSearchParams !== 'undefined')) {
  const urlParams = new URLSearchParams(window.location.search);
  tokenValue = urlParams.get('token');
}

const header = {
  "x-cassandra-token": tokenValue || "populate_me"
}
```